### PR TITLE
Improve Investment Details page

### PR DESCRIPTION
**Pull request by tyom**

### CODE REVIEW

File: src/views/_components/key-value-table.njk [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{##
+ # @param {object} [items] An object of items with label as key
+ # @param {string} [variant] A table
+ #
+ # @return {string} HTML for key/value table
+ #
+ # @example
+ #   {% component 'key-value-table', variant='striped' items={
```
Comment (teneightfive) [inline src/views/_components/key-value-table.njk]: Missing comma after second arg.

File: src/views/_components/key-value-table.njk [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ #     'Key #1: '80%',
+ #     'Key #2: '20%'
+ #   } %}
+ #}
+
+{% if items|length %}
+  <table class="table--key-value{{ ' table--' + variant if variant }}">
```
Comment (teneightfive) [inline src/views/_components/key-value-table.njk]: Does this really need to be a table? Doesn't feel like tabular data. 

Maybe a `dl` would be a more suitable element.

File: src/views/_components/project-status.njk [hunk 3]
```diff
@@ -1,15 +1,30 @@
-{% if project_code %}
+{##
+ # @param {string} [id] Project id
+ # @param {string} [name] Project name
+ # @param {string} [projectCode] *required* Project code
```
Comment (teneightfive) [inline src/views/_components/project-status.njk]: What's out pattern for documenting component variables?

I've previously used `(optional)` for data this isn't required and used the assumption everything else is required.

File: src/views/_components/project-status.njk [hunk 4]
```diff
@@ -1,15 +1,30 @@
...
+ # @param {string} [phaseName] Name of project phase
+ #
+ # @return {string} HTML for project status
+ #
+ # @example
+ #   {% component 'project-status', {
```
Comment (teneightfive) [inline src/views/_components/project-status.njk]: `project-status` feels like a content specific name for this component.

Maybe `status-bar` to match the class names?

File: config/nunjucks.js [hunk 5]
```diff
@@ -16,11 +16,12 @@ function ComponentExtension (env) {
     return new nodes.CallExtension(this, 'run', args)
   }
 
-  this.run = function run (context, name, locals = {}) {
+  this.run = function run (context, name, ...locals) {
```
Comment (teneightfive) [inline config/nunjucks.js]: If we're changing how we can call components we should document in the readme.

File: src/views/_components/key-value-table.njk (hunk 2 above)
Comment (tyom) [inline src/views/_components/key-value-table.njk]: Just moving to existing component structure. We can decide whether this should be a table in a different PR.

File: config/nunjucks.js (hunk 5 above)
Comment (tyom) [inline config/nunjucks.js]: Done.

**Comment by tyom** [comment]
Moving these changes in #94.
